tensorflow/core/kernels/mkl/mkl_matmul_ops_common.h

```diff
@@ -102,8 +102,10 @@ struct MklDnnMatMulFwdParams {
   struct PostOpParam {
     string name;
     std::vector<float> param;
+    string partial_key = string("");
```

Comment:
Is there a reason this is explicitly initialized?

Reply:
Removed explicit initialization.